tox.ini

```diff
@@ -78,8 +111,10 @@ commands =
     tests: coverage report
     functests: pytest {posargs:tests/functional/}
     bddtests: behave {posargs:tests/bdd/}
-    lint: prospector
-    lint: prospector tests
+    lint: pylint lms
+    lint: pylint --rcfile=pylintrc.tests tests/unit tests/functional
```

Comment:
It's necessary to run two separate pylint commands if we want to apply different linting rules to the code and tests